content/en/docs/ibm/pipelines.md

```diff
@@ -8,7 +8,7 @@ By default, Kubeflow Pipelines on IBM Cloud are running with the Tekton backend.
 
 You can also do a [standalone installation of Kubeflow Pipelines with Tekton](https://github.com/kubeflow/kfp-tekton/blob/master/guides/kfp_tekton_install.md#standalone-kubeflow-pipelines-with-tekton-backend-deployment) if you don't want whole of Kubeflow. 
 
-<img src="./kfp-tekton.png" alt="KFP-Tekton">
+<img src="../kfp-tekton.png" alt="KFP-Tekton">
```

Comment:
I don't see it working with this path, can you try with the below absolute path?
```suggestion
<img src="/docs/ibm/kfp-tekton.png" alt="KFP-Tekton">
```